docs/git/cheatsheet.md

```diff
@@ -0,0 +1,135 @@
+
+Visual Studio Code will now open with the code from the repo.
+
+## I want to save my code to the repo (Committing)
+
+Follow these steps if you have edited some code, and want to update the repo with your changes.
+
+#### Video
+
+<video width="100%" controls>
+  <source src={require('./assets/committing-video.mp4').default} />
+</video>
+
+#### Screenshots
+
+1. Open Github Desktop and ensure you are in the correct Current Repository
+2. A _diff_ showing the changes that you made is shown (lines with a green background were added, and lines with a red background were removed):
+
+![Showing a diff](./assets/committing-1.png)
+
+3. Click on the files in the sidebar to see different files. Make sure that you're only changing files that you meant to change
+4. Once you're happy with your change, fill out the input boxes in the bottom left to add a _commit message_:
+
+![Adding a commit message](./assets/committing-2.png)
+
+5. Click the "Commit to main" button
+
+You just committed your code to the repo! 🎉
+
+You should now see "No local changes". You can now make more changes to the repo (and more commits), or you can push your existing commits.
+
+## I want to send my code to volunteers (Pushing)
+
+Follow these steps if you have committed some code, and want to send it to the volunteers.
+
+#### Video
+
+<video width="100%" controls>
+  <source src={require('./assets/pushing-video.mp4').default} />
+</video>
+
+#### Screenshots
+
+1. Open Github Desktop and ensure you are in the correct Current Repository
+2. Click the "Push origin" button at the top:
+
+![Push origin](./assets/pushing-1.png)
+
+The next steps are different depending on whether this is the first time you have pushed this repo to Github. **If this is the first time you have pushed this repo, follow steps 3, 4 and 5 below. If not, you can skip those steps.**
+
+3. If this is the first time you have pushed this repo, a message pops up asking if you want to _fork_ this repository. Click the Fork this Repository button:
+
+![Fork this repository](./assets/pushing-2.png)
+
+4. A new message then pops up asking how you are planning to use the fork. Click the "To contribute to the parent project" option, then click Continue:
+
+![Contribute to the parent project](./assets/pushing-3.png)
+
+5. Click the "Push origin" button again
+
+6. In the menubar, click the Branch > Show Pull Request option:
+
+![Contribute to the parent project](./assets/pr-1.png)
+
+7. This will open a tab in your browser. Click the "Create pull request" button on the right:
+
+![Contribute to the parent project](./assets/pr-2.png)
+
+8. Fill out the Pull Request title (it is helpful to include your city, name and which week the coursework is for):
+
+![Contribute to the parent project](./assets/pr-3.png)
+
+9. Click the "Create pull request" button
+
+Congrats! You just created a Pull Request on Github! 🎉
+
+The volunteers will now be able to review your code and give feedback!
+
+## TODO
```

Comment:
Do we want to add a glossary?

Reply:
Right yes, I did mean to think about that.

Reply:
I've added it as a TODO.

@ChrisOwen101 @illicitonion do you have thoughts about just leaving these here as TODOs? If the intention is to get this out "in beta" for Chris' upcoming lesson, then we can come back to these later?

Reply:
Yeah, that's fine for me.

Maybe write it up to be a bit more "student friendly" e.g. `TODO` > `Coming Soon` (or something that effect
)